### ISSUE TITLE
-B hickups 

### ISSUE THREAD

**Issue by joa-quim** [GenericMappingTools/gmt#1018]
No no error but no TITLE
```
gmt psbasemap -R0/6/-1/1 -Jx2.5 -Bpa0g1+t"TITLE" -P > lixo.ps
```
errors
```
gmt psbasemap -R0/6/-1/1 -Jx2.5 -Ba0g1+t"TITLE" -P > lixo.ps
psbasemap [ERROR]: Syntax error -B option. Correct syntax:
        -B[p|s][x|y|z]<intervals>[+a<angle>|n|p][+l|L<label>][+p<prefix>][+u<unit>] -B[<axes>][+b][+g<fill>][+o<lon>/<lat>][+t<title>] OR
        -B[p|s][x|y|z][a|f|g]<tick>[m][l|p] -B[p|s][x|y|z][+l<label>][+p<prefix>][+u<unit>] -B[<axes>][+b][+g<fill>][+o<lon>/<lat>][+t<title>]
psbasemap [ERROR]: Offending option -Ba0g1+tTITLE
```
works
```
gmt psbasemap -R0/6/-1/1 -Jx2.5 -Ba0g1 -B+t"TITLE" -P > lixo.ps
```

**Comment by seisman** [comment]
Closed by #1021

**Comment by PaulWessel** [comment]
I thought these issues would auto-close if I write something like this PR closes #theissuenumber.  Am I not doing it right?

**Comment by seisman** [comment]
Yes, `closes #xxx` works, but you wrote `closes issue #xxx` in #1021.
